linker/Mono.Linker.Steps/MarkStep.cs

```diff
@@ -860,7 +866,9 @@ static bool IsDefaultConstructor (MethodDefinition method)
 			return IsConstructor (method) && !method.HasParameters;
 		}
 
-		static bool IsConstructor (MethodDefinition method)
+		protected static MethodPredicate IsConstructorPredicate = new MethodPredicate (IsConstructor);
```

Comment:
This seems redundant, compiler can do good job if you just use MethodGroupExpression for predicates